doc/source/whatsnew/v2.0.0.rst

```diff
@@ -659,6 +659,8 @@ If installed, we now require:
 +-------------------+-----------------+----------+---------+
 | python-dateutil   | 2.8.2           |    X     |    X    |
 +-------------------+-----------------+----------+---------+
+| tzdata            | 2022.1          |on Windows|    X    |
```

Comment:
This is misleading, since you'll need this package if you don't have the tz db on your computer, regardless of OS.

It's just that Windows always doesn't have it.

Also, I'm pretty sure I added tzdata as an optional dep a while back, so if you could remove that too, that'd be great.